pages/common/vim.md

```diff
@@ -14,6 +14,28 @@
 
 `view {{file}}`
 
-- Exit vim:
+- Save and Exit vim:
 
-`[Esc] (to switch to normal mode), then :q`
+`<Esc>:wq<Cr>`
```

Comment:
<Cr> is *very* obscure to the everyday computer user. I'd much rather we use <Enter>.